microsoft-edge/progressive-web-apps-chromium/how-to/app-actions.md

```diff
@@ -0,0 +1,402 @@
+---
+title: Enable App Actions on Windows for a PWA
+description: How to author and test an App Action for the App Actions on Windows framework, for a Progressive Web App (PWA) that you've created.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: pwa
+ms.date: 05/19/2025
+---
+# Enable App Actions on Windows for a PWA
+
+You can author and test an App Action for the App Actions on Windows framework, for a Progressive Web App (PWA) that you've created.  An App Action on Windows is an individual unit of behavior that a Windows app can implement and register.  The App Action can then be accessed from other apps and experiences, seamlessly integrating into user workflows.
+
+An app builds and registers each App Action, and then Windows apps or non-Windows apps can recommend registered App Actions to the user at contextually relevant times and locations within the user's workflow.
```

Comment:
"Windows apps or non-Windows apps" feels a bit odd to me. What's a non-Windows app?
What about this instead:

```suggestion
An app builds and registers each App Action, and then Windows apps and experiences can recommend registered App Actions to the user at contextually relevant times and locations within the user's workflow.
```